Alchemy consolidation

WeNeedMorePhoron opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->

## About The Pull Request
- Merges the two alchemy skills together, favoring "Alchemy" instead of "Lesser Alchemy"
- Stone & Alchemical mortar has been merged. Renamed alchemical mortar, used the old description, used the "new" icon. Delted old icon
- Skills on all classes has been merged, this has some "balance" changes because some classes were missing one or the other (mostly the new alchemy). 
- On **request**, alchemy has been removed from merchant. They had 4 (!!!) Expert in lesser alchemy (And now potion) and that is not part of their job profile.


<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game
No more double alchemy skills, no more inconsistency like adv mage having both alchemy but magician associate having two in lesser and none in potion alchemy.

## Classes Changes
Here's a list of all classes who got actual changes (I.e. they didn't have both alchemy). This does result in a bigger proliferation of potion academy but most of them are niche / sensical classes anyway.

- Lich got 5 in Alchemy now instead of only Lesser
- Rogue mage has 3 in Both Alchemy
- Barber Surgeon now have 2 in both alchemy
- Druid now have 3 in both alchemy (Questionable, but I am not balancejakking)
- Desert Rider now have 2 in both alchemy
- Underdweller have 2 in both alchemy
- Hierophant / Vizier have 3 / 2 in both alchemy
- Hand now have 4 in both alchemy
- Soilson have 1 in both alchemy
- Alchemist (not used) gets both alchemy
- Archivist now have 6 in alchemy (Previously, 6 in lesser, 4 in potion)
- Magician Associate now have 3 in alchemy
- Mage Apprentice now have 2 in alchemy
- **ON REQUEST, Merchant has their 4 in alchemy removed.**

## Testing 
I've checked that the crafting recipes got collapsed, grinded Ozium and Moondust and mineral dust and seeds in the "new" mortar, and made salt piles and meat. 

I might've missed some things because alchemy is big and I made a big snowflake mess of code. I'll follow up and fix any issues I found

<!-- Argue for the merits of your changes and how they benefit the game, especially if they are controversial and/or far reaching. If you can't actually explain WHY what you are doing will improve the game, then it probably isn't good for the game in the first place. -->

WeNeedMorePhoron commented:
Yep the two mortar functionality were merged in. I hope I didn't miss anything

WeNeedMorePhoron commented:
des scholar and failed apprentice are both there yes. I looked for every instance and changed them.

MothionTechnologies commented:
What about the Merchant? They have a 4 in lesser alchemy at round start.

WeNeedMorePhoron commented:
Missed it, but they got it swapped to normal alchemy

Cleric2145 commented:
Could you perhaps use the simple mortar recipe for crafting? Requiring iron ore is a big hurdle for the more casual grinding needs, and I don't feel like the mortar gives a big enough benefit to require ore.

WeNeedMorePhoron commented:
Good catch - I removed the "new" alchemical mortar recipe I forgot to delete (there was two of them) and renamed the old one (That needs 1 stone) to "alchemical mortar".

Rendsew commented:
Grinding things into dust within mortars does not appear to be working correctly. Attempted to grind ozium, westleach and swampweed and all are being devoured and turned into their chemical forms rather than the dust form, with the chemical form not actually being all that useful for alchemy except in the case of poison berries. I'm guessing its cause they HAVE chemical recipes attached, since everything without grinds just fine.

WeNeedMorePhoron commented:
This PR is ready to go and resulted in much more alchemy in general but most players are fine with it from what I see. Maybe further tweaking is needed later on.
